### PR TITLE
[SMALLFIX] Replace with Map.getOrDefault

### PR DESCRIPTION
**Pull request by wulinlide**
in alluxio/job/common/src/main/java/alluxio/job/util/TimeSeries.java

### CODE REVIEW

File: job/common/src/main/java/alluxio/job/util/TimeSeries.java [hunk 1]
```diff
@@ -80,8 +80,7 @@ public void record(long timeNano) {
    */
   public void record(long timeNano, int numEvents) {
     long leftEndPoint = bucket(timeNano);
-    mSeries.put(leftEndPoint,
-        (mSeries.containsKey(leftEndPoint) ? mSeries.get(leftEndPoint) : 0) + numEvents);
+    mSeries.put(leftEndPoint, (mSeries.getOrDefault(leftEndPoint, 0)) + numEvents);
```
Comment (aaudiber) [inline job/common/src/main/java/alluxio/job/util/TimeSeries.java]: You can remove the parentheses around `mSeries.getOrDefault(leftEndPoint, 0)` - they are no longer needed

```java
mSeries.put(leftEndPoint, mSeries.getOrDefault(leftEndPoint, 0) + numEvents);
```

Comment (wulinlide) [reply job/common/src/main/java/alluxio/job/util/TimeSeries.java]: Thanks for your advice.

**Comment by wulinlide** [comment]
alluxio-bot, check this please

**Comment by aaudiber** [comment]
alluxio-bot, merge this please

**Pull request merged**